code/_onclick/item_attack.dm

```diff
@@ -84,6 +88,12 @@ avoid code duplication. This includes items that may sometimes act as a standard
 		return 0
 
 	var/hit_zone = M.resolve_item_attack(src, user, target_zone)
+
+	var/datum/attack_result/AR = hit_zone
+	if(istype(AR))
+		if(AR.hit_zone)
+			apply_hit_effect(AR.attackee ? AR.attackee : M, user, AR.hit_zone)
```

Comment:
`AR.attackee || M`

Reply:
done, also fixed a runtime that showed up on audio